data/brands/leisure/fitness_centre.json

```
@@ -13,6 +13,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Econofitness",
```

I don't see a reason to remove the accent mark in the name.
```suggestion
      "displayName": "Éconofitness",
```